stencil-library/src/components/details/details.tsx

```diff
@@ -0,0 +1,97 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+import { ErrorState } from './utils';
+
+interface IEntityProps {
+  title: string;
+  entityHeadInfo: {
+    title: string,
+    value: any
+  }[];
+  entitySections: {
+    sectionTitle: string;
+    sectionDetails: {
+      title: string,
+      value: any
+    }[];
+  }[];
+  metadata: any;
+}
+
+@Component({
+  tag: 'justifi-details',
+  styleUrl: './details.scss',
+  shadow: true,
+})
+
+export class Details {
+  @Prop() errorMessage: string;
+  @Prop() entity: IEntityProps;
+
+  render() {
+    return (
+      <Host
+        exportParts='detail-loading-spinner,detail-loading-state,detail-empty-state,
+        detail-head,detail-title,detail-method,detail-info,detail-info-item,
+        detail-info-item-title,detail-info-item-data,detail-metadata,detail-metadata-title,
+        detail-method-title,detail-method-data'
+      >
+        {
+        this.errorMessage ?
+          ErrorState(this.errorMessage)
+        :
+          <main class="p-2">
+            <div part='detail-head' class="p-2">
+              <div class="d-flex flex-row align-items-center gap-2 mb-2">
+                <h1 class="m-0 text-uppercase" part='detail-title'>{this.entity.title}</h1>
```

Comment:
I think this probably should not be an h1, but if you need the text to be the size of an h1 maybe there is a bootstrap utility class for font size? I also think it should not be `text-uppercase`, but instead the text passed should be uppercase when appropriate (like the USD in your screenshot)

Reply:
About it being an `<h1>`, it's in the design, but I'm happy to change it. Do you think it might be an issue for screen readers?

About the uppercase, I agree, I can uppercase the text when passing it. I'll change that.